andela/ah-backend-targaryen
# #160609549 generate jwt token

### What does this PR do?

Creates a JWT token on signup and login

### Description of tasks to be completed?

Currently, users do not receive an authentication token when they register an account

This task adds a JWT token to be received by users when the register and when they login such that they can be able to use system features.

### How should this be manually tested?

Get the deployment link on heroku (Pasted below for convenience.)
https://ah-backend-targaryen-sta-pr-14.herokuapp.com
Open postman, paste the link, and add relevant login url `/api/users/login/`. Provide the username, email, and password in the body in the format stated below.
```source-json
{
	"user": {
		"email": "davidm@jake.jake",
		"password": "1234567890"
	}
}
```
You should be able to receive a JWT token on successful registration.

### What are the relevant pivotal tracker stories?

[#160609549](https://www.pivotaltracker.com/story/show/160609549)

### Any background context you want to add?

N/A

### Screenshots
<img width="1127" alt="screen shot 2018-10-01 at 10 41 01" src="https://user-images.githubusercontent.com/38077368/46275708-8b58f880-c566-11e8-91aa-c835fa17f3b6.png">
